src/runner_manager.py

```diff
@@ -193,14 +190,13 @@ def get_latest_runner_bin_url(
         logger.debug("Response of runner binary list: %s", runner_bins)
 
         try:
+            arch = self.config.charm_state.arch.value
```

Comment:
separate variable assignment to remove "Line break occurred before a binary operator (W503) (https://www.flake8rules.com/rules/W503.html)" error and "Line too long" error.